subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -202,15 +202,18 @@ int lwm2m_engine_connection_resume(struct lwm2m_ctx *client_ctx)
 	int ret;
 
 	if (client_ctx->connection_suspended) {
-		lwm2m_close_socket(client_ctx);
-		client_ctx->connection_suspended = false;
-		ret = lwm2m_open_socket(client_ctx);
-		if (ret) {
-			return ret;
+		if (IS_ENABLED(LWM2M_RD_CLIENT_STOP_POLLING_AT_IDLE)) {
```

Comment:
Aren't we missing `CONFIG_` prefix here?

Reply:
Oh.. right.. How did I miss that.. I'll fix.